dev-util/quilt/quilt-0.66.ebuild

```diff
@@ -33,7 +33,7 @@ pkg_setup() {
 
 src_prepare() {
 	# Add support for USE=graphviz
-	use graphviz || eapply -p0 "${FILESDIR}/${P}-no-graphviz.patch"
+	use graphviz || eapply "${FILESDIR}/${P}-no-graphviz.patch"
```

Comment:
I think a better version is:
`use graphviz || PATCHES+=( "${FILESDIR}"/${PN}-0.66-no-graphviz.patch )`
(I think `${PN}-0.66` is better than `${P}` for future versions can easier reuse, and the version in the patch name represents the first version in which it is used)

Reply:
true that. Also, just realized my in-use ::gentoo tree was out of date relative to my dev tree,
so this had already been fixed.